### PR TITLE
Add false positive test package to detect errors in elastic agent logs

### LINKED ISSUE #1811: Add negative tests for detection of issues in elastic agent logs
`elastic-agent` supports finding issues in Elastic Agent logs. We have had some regressions and fixes around there, that could have been detected earlier by negative tests.

We should find a test case that can be used for negative tests. One challenge for this is that these issues use to be related to specific versions of the stack, and we always test now with the default version of the stack.

For example we know that prometheus package had issues in 8.12.0, but these issues were caused by a regression in the stack.

### PR DESCRIPTION
**Pull request by mrodm**
Closes #1811 

Added a new test package into `false_positives` to be able to check if there is any error in the logs from Elastic Agent container.

This package requires to be executed in a specific Elastic stack version 8.12.0 in order to reproduce the error.

Tested failure (not detected error logs) build (using 8.12.1):
- https://buildkite.com/elastic/elastic-package/builds/3958#0191c69d-8818-4b98-95b1-34fcface0b97

### CODE REVIEW

File: scripts/test-check-false-positives.sh [hunk 1]
```diff
@@ -59,6 +59,11 @@ function check_expected_errors() {
     cat ${results_no_spaces} | grep -E "${line}"
   done < "${expected_errors_file}"
 
+  # Copy XML files to another extension so they are not used to check jUnit tests
+  # but those files will be able to be reviewed afterwards
+  for file in $(ls $result_tests) ; do
+      cp "${file}" "${file}.txt"
```
Comment (jsoriano) [inline scripts/test-check-false-positives.sh:65]: Nit. Could we add some meaningful suffix?
```suggestion
      cp "${file}" "${file}-expected-errors.txt"
```

File: scripts/test-check-false-positives.sh [hunk 2]
```diff
@@ -59,6 +59,11 @@ function check_expected_errors() {
     cat ${results_no_spaces} | grep -E "${line}"
   done < "${expected_errors_file}"
 
+  # Copy XML files to another extension so they are not used to check jUnit tests
+  # but those files will be able to be reviewed afterwards
```
Comment (jsoriano) [inline scripts/test-check-false-positives.sh:63]: :+1:

**Review by jsoriano — COMMENTED** [review]
Nice, thanks!

File: scripts/test-check-false-positives.sh (hunk 1 above)
Comment (mrodm) [reply scripts/test-check-false-positives.sh:65]: Sure!
I'll update it.

**Review by jsoriano — APPROVED** [review]

**Pull request merged**